packages/web/src/utils/theme/theme.ts

```diff
@@ -1,31 +1,13 @@
-import { Theme } from '@audius/common/models'
+import { SystemAppearance, Theme } from '@audius/common/models'
 
 const THEME_KEY = 'theme'
 export const PREFERS_DARK_MEDIA_QUERY = '(prefers-color-scheme: dark)'
 
-const getThemeNameKey = (theme: Theme) => {
-  switch (theme) {
-    case Theme.AUTO:
-      if (doesPreferDarkMode()) {
-        return Theme.DARK
-      }
-      return Theme.DEFAULT
-    case Theme.DEFAULT:
-    case Theme.DARK:
-    case Theme.MATRIX:
-    default:
-      return theme
-  }
-}
-
-const applyTheme = (theme: Theme) => {
-  // Set data-theme to enable theme scoped css rules
-  document.documentElement.setAttribute('data-theme', getThemeNameKey(theme))
-}
-
 export const doesPreferDarkMode = () => {
   return (
-    window.matchMedia && window.matchMedia(PREFERS_DARK_MEDIA_QUERY).matches
+    typeof window !== 'undefined' &&
```

Comment:
nit: Is there any particular reason for the `typeof` rather than just `window !== undefined`? or even just a falsy check?

Reply:
not sure actually, but this is what i see for ssr checks generally, @sliptype thoughts?